src/NetMQ/NetMQSocket.cs

```diff
@@ -400,6 +400,19 @@ public void Monitor([NotNull] string endpoint, SocketEvents events = SocketEvent
             m_socketHandle.Monitor(endpoint, events);
         }
 
+        /// <summary>
+        /// stop monitoring all enpoints for SocketEvent events.
+        /// </summary>
+        /// <exception cref="ObjectDisposedException">This object is already disposed.</exception>
+        /// <exception cref="TerminatingException">The socket has been stopped.</exception>
+        /// <exception cref="NetMQException">on failure to stop socket.</exception>
+        public void StopMonitor()
+        {
+            m_socketHandle.CheckDisposed();
+
+            m_socketHandle.Monitor(null, SocketEvents.All);
+        }
+
```

Comment:
It is could be possible with next restructuring update from netMQ, since they are moving things into different packages now. There is a issue filed for monitoring and next migration should probably happen when that issue is merged into restructured netMQ